test/spec/features/modeling/behavior/ConnectEventBasedGatewayBehaviorSpec.js

```diff
@@ -0,0 +1,36 @@
+import {
+  bootstrapModeler,
+  inject
+} from 'test/TestHelper';
+
+import modelingModule from 'lib/features/modeling';
+import coreModule from 'lib/core';
+
+
+describe('features/modeling/behavior - connect event based gateway to receive task with boundary', function() {
+
+  var testModules = [ coreModule, modelingModule ];
+
+  var diagramXML = require('./ConnectEventBasedGatewayBehavior.bpmn');
+
+  beforeEach(bootstrapModeler(diagramXML, { modules: testModules.concat(modelingModule) }));
```

Comment:
Why do we add `modelingModule` second time here? The tests seem to work without it.